docs/manual.md

```diff
@@ -538,48 +586,56 @@ Next$1 == year - 80 >= 18 /\ hasLicense' <- TRUE /\ (year' <- year)
 
 As you can see, the model checker did two things:
 
- 1. It has translated several expressions that look like `x' = e` into `x' <- e`.
-   For instance, you can see `year' <- 80` and `hasLicense' <- FALSE` in `Init$0`.
-   We call these expressions **assignments**.
-
- 1. It has replaced the operator `Next` into two operators `Next$0` and `Next$1`.
+1. It has translated several expressions that look like `x' = e` into `x' <- e`.
+   For instance, you can see `year' <- 80` and `hasLicense' <- FALSE` in
+   `Init$0`. We call these expressions **assignments**.
+1. It has factored the operator `Next` into two operators `Next$0` and `Next$1`.
    We call these operators **symbolic transitions**.
 
-The pure TLA+ does not have the notions of assignments and symbolic
+Pure TLA+ does not have the notions of assignments and symbolic
 transitions.  However, TLC sometimes treats expressions `x' = e` and `x' \in S`
 as if they were assigning a value to the variable `x'`. TLC does so
 dynamically, during the breadth-first search. Apalache looks statically for assignments
 among the expressions `x' = e` and `x' \in S`.
 
+<!-- TODO(shonfeder): The first stentence of the next passage isn't clear to me.
```

Comment:
@konnov or @andrey-kuprianov for my own edification, I'm hoping to get a bit of clarification on this point.

Reply:
Now that I'm reading the theory, I think I understand this part now. It is a restatement of the second point from above,  where we currently have "it has factored the operator...", right? What "factoring the operator into n operators" consists in is replacing the single operator `Op` into a disjunction of `n` operators `Op$0 \/ Op$1 \/ ... Op$n`. But what that looks like in terms of concrete syntax is just

```
Op$0 == ...

Op$1 == ...

...

Op$n == ...
```

Is this the correct understanding?

Reply:
I reworked my suggestion here, following the reasoning in this thread.

Reply:
There are two things about the assignment finder. First, it finds expressions that can be used as assignments. Having a complete set of assignments, the assignment finder decomposes transition predicates, e.g., `Init` and `Next` into disjunctions, as you pointed out.